travis.sh

```diff
@@ -67,6 +67,59 @@ build_java_oracle8() {
   build_java
 }
 
+build_android() {
```

Comment:
I wonder if it's worth using [apt-fast](https://github.com/ilikenwf/apt-fast) - it's supposed to be 26x faster than apt-get, and looks like it's just a shell script that performs the downloads in parallel..